src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -164,7 +164,7 @@ public FormulaValue GetResult(IRContext irContext)
             }
         }
 
-        private class MinDateTimeAgg : IAggregator
+        private class MinDateAndDateTimeAgg : IAggregator
```

Comment:
>MinDateAndDateTimeAgg [](http://example.com/codeflow?start=22&length=21)

Merged the aggregator for date and datetime values, since the logic (and underlying data type, DateTime) is the same.